src/components/nav-tabs/ODH.js

```diff
@@ -0,0 +1,191 @@
+import * as React from "react";
+import { useState } from 'react';
+import '../../pages/style.css';
+// Component imports
+import { Link } from "gatsby"
+import { Container, Text, Title, Group } from '@mantine/core';
+import { createStyles, Card, SimpleGrid, Anchor, UnstyledButton, Tooltip, Select } from '@mantine/core';
+// Service SVGs
+import JupyterLogo from "../../assets/services/jupyterhub.svg"
+import ArgoLogo from "../../assets/services/argo.svg";
+import SupersetLogo from "../../assets/services/superset.svg";
+import PrometheusLogo from "../../assets/services/prometheus.svg";
+import GrafanaLogo from "../../assets/services/grafana.svg";
+import SparkLogo from "../../assets/services/spark.svg";
+import SeldonLogo from "../../assets/services/seldon.svg";
+import KafkaLogo from "../../assets/services/kafka.svg";
+import AirflowLogo from "../../assets/services/airflow.svg";
+import HueLogo from "../../assets/services/hue.svg";
+
+const clusters = [
```

Comment:
@bryanmontalvan can we get `clusters` this file moved out as it's own `.js` file that we can update ?